yarn.lock

```diff
@@ -3152,16 +3152,6 @@
     regexpp "^3.0.0"
     tsutils "^3.17.1"
 
-"@typescript-eslint/eslint-plugin@2.31.0":
```

Comment:
Just wondering why we have any changes here as I cannot see corresponding package.json changes anywhere? Was yarn.lock not updated by a prior PR?

Reply:
:thinking: I don't know. I'll merge changes from canary.